text/machine_comprehension/bert-squad/README.md

```diff
@@ -0,0 +1,111 @@
+# BERT-Squad
+
+## Use cases
+This model uses masked language method to enable training of bidirectional models that can quickly and efficiently run through language processing tasks. The model also adds a next sentence prediction task to improve sentence-level understanding which helps to answer questions based on the given paragraph input. 
+
+## Description
+BERT (Bidirectional Encoder Representations from Transformers) applies Transformers, a popular attention model, to language modelling. This mechanism has an encoder to read the input text and a decoder that produces a prediction for the task. This model uses the technique of masking out some of the words in the input and then condition each word bidirectionally to predict the masked words. BERT also learns to model relationships between sentences, predicts if the sentences are connected or not. 
+
+## Model
+
+ |Model        |Download  |Checksum|Download (with sample test data)| ONNX version |Opset version|Dependencies| 
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
+|BERT-Squad| [393,930 KB](model/bert.onnx)  |dc59794145b4a37995a2667370dc3d6f |  [78 KB](model/saved_model.pb.gz) |  1.5.2  | 8| [tokenization.py](dependencies/tokenization.py) [run_onnx_squad.py](dependencies/run_onnx_squad.py)  | 
+
+## Inference
+We used onnxruntime to preform the inference. 
+
+### Input 
+Context paragraph and questions about the paragraph that are written in an inputs.json file. 
```

Comment:
Input should describe the input the actual model takes which are probably tensors that represent the the context paragraph and questions. The inputs.json is just how the preprocessing code we are using works. If someone used this model they would most likely not use an inputs.json, they would feed in the data directly.